src/components/AdaptiveTabs/AdaptiveTabs.tsx

```diff
@@ -64,16 +64,11 @@ export interface TabProps {
 
 // https://github.com/gravity-ui/components/issues/7
 class Tab extends React.Component<TabProps> {
-    onClick = (event: React.MouseEvent) => {
-        if (this.props.onClick) {
-            this.props.onClick(this.props.id, event);
-        }
-    };
-
     render() {
         const {active, disabled, hint, title = this.props.id} = this.props;
         const stringTitle = (hint || typeof title === 'string' ? title : '') as string;
         return (
+            // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
Why did you turn off the eslint rule instead of adding attributes for accessibility?